benchmarks/kernels/tuning_utils.py

```diff
@@ -0,0 +1,132 @@
+## Utilize method from rocm/Triton tuning script
+def get_full_tuning_space():
+    configs = []
+
+    block_mn_range = [16, 32, 64, 128, 256]
+    block_k_range = [16, 32, 64, 128, 256]
+    # split_k_range = [1] #, 2, 4, 5, 6, 8, 10, 12, 16, 18, 24]
+    num_warps_range = [1, 2, 4, 8]
+    group_m_range = [1, 4, 8, 16, 32]
+    # For now we see better perf with num_stages=0 for all gemm configs we care
+    # But keep this explicit so that we do not forget we may need to set it to
+    # other values in the future
+    num_stage_range = [0]
+    waves_per_eu_range = [0]
+    matrix_instr_nonkdim_range = [16, 32]
+    kpack_range = [1, 2]
+
+    for block_m in block_mn_range:
```

Comment:
probably best to use something like
`for block_m, block_n, ... in list(itertools.product(block_mn_range, block_mn_range, ...))`
for readanility

Reply:
Updated